### ISSUE TITLE
Using LinearRegressionOutput in a neural network

### ISSUE THREAD

**Issue by MikeHuYimin** [dmlc/mxnet#2773]
Currently I am dealing with a regression task to predict mulltiple value. I built a network as below:
 data <- mx.symbol.Variable("data")
 fc1 <- mx.symbol.FullyConnected(data, num_hidden = 500)
 act1 <- mx.symbol.Activation(fc1,act.type='tanh')
 fc2 <- mx.symbol.FullyConnected(act1, num_hidden = 500)
 act2 <- mx.symbol.Activation(fc2,act.type='tanh')
 fc3 <- mx.symbol.FullyConnected(act2, num_hidden = 500)
 act3 <- mx.symbol.Activation(fc3,act.type='tanh')
 fc4 <- mx.symbol.FullyConnected(act3, num_hidden = 11350) 
 out <- mx.symbol.LinearRegressionOutput(data = fc4)
 model <- mx.model.FeedForward.create(
  X = train.x,(970 times 100,column for samples)
  y = train.y, (11350 times 100)
  symbol = out,
  num.round = 100,
  learning.rate = 0.01,
  momentum = 0.9,
  eval.metric = mx.metric.rmse,
  array.batch.size = 10,
  array.layout = "colmajor"
)
The thing is that the output are all the same for all sample. And when I choose fc4 to be one node and try to predict each variable(any one in 11350 total), the result will be just fine, but when I predict all the value together, it will be totally the same output for different samples. If I choose to predict two varible together, one will be fine and the other will be the same, if I choose 3 or more, all the output will be the same.

The problem is that can I predict all the variables together, or just one at a time?
Is there anything wrong with my code?
Thanks a lot!

**Comment by MikeHuYimin** [comment]
I guess is I figure out the problem: the network itself is not capable of predict so much output at a time.
